lib/puppet/parser/functions/new.rb

```diff
@@ -153,6 +153,278 @@
 $a_number = Numeric("3.14")  # results in 3.14 (a float)
 ```
 
+Conversion to Timespan
+-------------------
+
+A new `Timespan` can be created from `Integer`, `Float`, `String`, and `Hash` values. Several variants of the constructor are provided.
+
+#### Timespan from seconds
+
+```puppet
+function Timespan.new(
+  Variant[Float, Integer] $value
```

Comment:
Since Float is supported, does the decimal part denote fractions of a second? (Documentation should say).